packages/runtime/test/src/parser/parser.ts

```diff
@@ -1266,15 +1367,26 @@ describe("Parser", function () {
       interpreter.tealVersion = 2;
     });
 
+    it("Supported pragma version 6", () => {
+      const fileWithPragmav6 = "test-pragma-v6.teal";
+      assert.doesNotThrow(() =>
+        parser(getProgram(fileWithPragmav6), ExecutionMode.SIGNATURE, interpreter)
+      );
+    });
```

Comment:
Also maybe add a failing test (`#pragma version 7`)?

Reply:
Thank, this is good test.